boards/arm/stm32_blackpill_v2/doc/index.rst

```diff
@@ -0,0 +1,163 @@
+.. _stm32_blackpill_v2:
+
+WeAct Studios Black Pill V2.0
```

Comment:
Board name provided as the doc title should match the one provided in board yaml file:
`WeAct Black Pill V2.0`

Reply:
> Board name provided as the doc title should match the one provided in board yaml file:
> `WeAct Black Pill V2.0`

Fixed.